src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -302,14 +287,14 @@ export function SavedObjectProvider(esAdmin, kbnIndex, Promise, Private, Notifie
      * @resolved {String} - The id of the doc
```

Comment:
Does this still resolve to the id?  According to the change below, it looks like it now returns a response object with an id property on it.